scripts/release

```diff
@@ -2,7 +2,8 @@
 # This script updates lepidopter version and ChangeLog files
 set -e
 
-VERSION_FILE="lepidopter-fh/etc/lepidopter_version"
+source conf/lepidopter-image.conf
+UPDATE_INCREMENT=$(awk '{print $1+1}' ${UPDATE_VERSION_FILE})
```

Comment:
I don't see where is that diff you are actually writing what is the latest version number. From the looks of it you are just downloading the updater and the update files, but not the actual version number (you are excluding `latest_version`)